Traceback (most recent call last):
  File "/home/alex/.local/lib/python3.10/site-packages/concrete/fhe/compilation/compiler.py", line 447, in compile
    self._evaluate("Compiling", inputset)
  File "/home/alex/.local/lib/python3.10/site-packages/concrete/fhe/compilation/compiler.py", line 292, in _evaluate
    self._trace(first_sample)
  File "/home/alex/.local/lib/python3.10/site-packages/concrete/fhe/compilation/compiler.py", line 220, in _trace
    self.graph = Tracer.trace(self.function, parameters)
  File "/home/alex/.local/lib/python3.10/site-packages/concrete/fhe/tracing/tracer.py", line 79, in trace
    output_tracers: Any = function(**arguments)
  File "<user_code>", line 5, in circuit
NameError: name 'np' is not defined
